openmmtools/forces.py

```diff
@@ -254,14 +254,22 @@ class RadiallySymmetricRestraintForce(utils.RestorableOpenMMObject):
         The indices of the first group of atoms to restrain.
     restrained_atom_indices2 : iterable of int
         The indices of the second group of atoms to restrain.
+    controlling_parameter_name : str
+        The name of the global parameter controlling the energy function.
     *args, **kwargs
         Parameters to pass to the super constructor.
 
+    Attributes
+    ----------
+    controlling_parameter_name
+
     """
 
     def __init__(self, restraint_parameters, restrained_atom_indices1,
-                 restrained_atom_indices2, *args, **kwargs):
+                 restrained_atom_indices2, controlling_parameter_name,
```

Comment:
Can we make `controlling_parameter_name` have a default? E.g. `controlling_parameter_name = 'lambda_restraints'`? Or would we instead just want `controlling_parameter_name = 'restraints'` and the `lambda_` prefix is assumed prepended?